src/lib/viewers/error/PreviewErrorViewer.js

```diff
@@ -111,17 +117,31 @@ class PreviewErrorViewer extends BaseViewer {
     }
 
     /**
-     * Adds optional download button
+     * Adds a link button underneath error message.
+     *
+     * @param {string} linkText - Button message
+     * @param {string} linkUrl - URL for link
+     * @return {void}
+     */
+    addLinkButton(linkText, linkUrl) {
+        const linkBtnEl = this.infoEl.appendChild(document.createElement('a'));
+        linkBtnEl.className = 'bp-btn bp-btn-primary';
+        linkBtnEl.target = '_blank';
+        linkBtnEl.textContent = linkText;
```

Comment:
Should this be localized too?

Reply:
linkText will be localized before we pass it in.